src/backend/aspen/workflows/pangolin/save.py

```diff
@@ -15,14 +15,19 @@
 from aspen.database.models import PathogenGenome, UploadedPathogenGenome
 
 
-def get_probability(ambiguity_score: float) -> int:
+def get_probability(row: dict) -> Union[int, None]:
     """
     Estimate confidence percentage based on the `ambiguity_score`.
     Per pangolin docs, this is basically the number of lineage-defining sites
     that had to be imputed from the reference sequence.
     Round and multiply by 100 --> percentage for easier user comprehension.
     """
-    return round(ambiguity_score * 100)
+    if row["ambiguity_score"] != "":
+        return round(row["ambiguity_score"] * 100)
+    elif "Assigned using designation hash" in row["note"]:
+        return 100
+    else:
+        return None
```

Comment:
could just be:
```
elif "Assigned using designation hash" in row["note"]:
        return 100
 return None
```

Reply:
true! this is just a personal preference thing / I tend to make these things explicit. But if it's better practice, happy to make the change :)